src/website/src/pages/ec/components/list/docs/usage.md

```diff
@@ -0,0 +1,151 @@
+---
+title: Usage
+order: 1
+---
+
+import { Paragraph } from '@ecl/website-components';
+
+<Paragraph size="lead">
+Lists group together related pieces of information so they are clearly associated with each other and easy to read.
+Lists are good from a structural point of view as they help create a well-structured and more accessible.
+Semantically correct lists help visitors read your web site.
+</Paragraph>
+
+## Anatomy
+
+Lists or "typography lists" are optimized for reading comprehension.
+A list consists of a single continuous column of subdivisions called rows that contain items.
+Content of each item may be plain text, or other elements (links, paragraphs, ...)
+A list should have a contextual coherence and allow the user to understand the message instantly.
+
+### Nesting lists
+
+It is perfectly ok to nest one list (of any type) inside another one.
+You might want to have some sub-bullets sitting below a top level bullet.
+Just make sure that nested items are all related to their parent.
+
+Also try to keep your nested lists under control; having 4 or 5 nested level may create confusion and is generally a sign that the content has to be structured differently (break down into several lists for instance).
+
+### Three different types of lists
+
+1. unordered list
```

Comment:
this is not rendered correctly on the website (probably due to the extra lines)
![image](https://user-images.githubusercontent.com/25579221/53497336-3d5c0880-3aa4-11e9-9c64-8858720ef7c3.png)

Reply:
yup